console/algorithms/Cargo.toml

```diff
@@ -53,3 +53,7 @@ version = "1.0"
 [dev-dependencies.serde_json]
 version = "1.0"
 features = [ "preserve_order" ]
+
+[features]
+default = [ ]
+parallel = [ "snarkvm-fields/parallel" ]
```

Comment:
Commenting again to re-enforce the above comment, I doubt it'll be passable to remove parallel from the default features, but we can look for another solution.